WcaOnRails/app/views/incidents/index.html.erb

```diff
@@ -0,0 +1,108 @@
+<% provide(:title, "Incidents log") %>
+
+<%= render layout: "nav" do %>
+  <h1><%= yield(:title) %></h1>
+
+  <div id="custom-search-toolbar">
+    <input id="incident-tags" class="form-control" type="text" placeholder="Filter by tags">
+  </div>
+  <div class="incidents-log-table-container">
+    <%= wca_table(table_class: "incidents-log-table", data: { toggle: "table", pagination: true, search: true, toolbar: "#custom-search-toolbar" }, floatThead: false, greedy: false) do %>
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th class="tags-cell">Tags</th>
+          <th class="comps-cell">Happened during</th>
+          <th class="status-cell">Status</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <% @incidents.each do |incident| %>
+          <tr>
+            <td><%= link_to incident.name, incident %></td>
+            <td data-tags="<%= incident.tags %>">
+              <% incident.tags_array.each do |t| %>
+                <%= render "incident_tag", tag: t %>
+              <% end %>
+            </td>
+            <td data-tags="<%= incident.competitions %>">
+              <% incident.competitions.each do |c| %>
+                <%= render "competition_tag", comp: c %>
+              <% end %>
+            </td>
+            <td><%= incident.status %></td>
+          </tr>
+        <% end %>
+      </tbody>
+    <% end %>
+  </div>
+
+  <script>
+var resetSelectedTags = function(e, tag) {
+  e.preventDefault();
+  $('.search input').val("");
+  var selectize = $('input#incident-tags')[0].selectize;
+  selectize.clear();
+  selectize.addItem(tag);
+}
+
+
+var bootstrap_object = null;
+// Can't use filterBy, as some cells have multiple value
+var search_func = function(text) {
```

Comment:
uber nit: search_func -> searchFunc, because we use camelCase in javascript.

Reply:
Also, good to keep in mind that (as mentioned before) at some point it would be better to use the server side searching (as we do for most bootstrap-tables) and this whole logic would be unnecessary. I would consider making this change now (especially that the JS code is **really** tricky, as it needs to be done anyway. @jfly thoughts?

Reply:
Do you think that switching to server side searching would make this code less tricky?

The WRC currently uses a google spreadsheet to manage all incidents, and that all gets loaded into client side memory (at least, I think that's how google spreadsheets work). @viroulep, do you have a sense of how many incidents there will be? If it's in the hundreds, I think it's totally fine doing it client side. If it's in the low thousands, I think it's probably still ok? If it's in the tens of thousands, then yeah, this should probably be server side.

Reply:
Searching would be way easier (I believe it would be a quite simple query). On the other hand, I've just realised that we would need to handle pagination, even though it's not that much pain it's still hard to estimate how better this would look.

Reply:
I think it's definitely going to be several hundreds.
This log is supposed to have only "worthy" incidents, basically those not fully covered by the Regulations, so I don't think we'll have thousands of them.

We started the spreadsheet roughly one year ago and we have 54 incidents recorded there.